examples/two_dimensional_mountain_wave.jl

```diff
@@ -0,0 +1,386 @@
+# ```
+#
+# Waves propagate vertically when ``m^2 > 0`` (i.e., ``k < k^*``), and decay exponentially
+# when ``m^2 < 0`` (i.e., ``k > k^*``), where the critical wavenumber is:
+#
+# ```math
+# k^* = \sqrt{\frac{N^2}{U^2} - \frac{\beta^2}{4}}
+# ```
+#
+# !!! note "Current limitations"
+#     This validation case requires high resolution to properly resolve the low-elevation
+#     terrain corrugations with the immersed boundary method. Additionally, open lateral
+#     boundary conditions have not been implemented; periodic boundaries are used instead,
+#     which is not ideal for this test case.
+
+using Breeze
+using Oceananigans.Units
+using Printf
+using CairoMakie
+
+using Oceananigans.Grids: znode, xnodes
+using Oceananigans: Face, Center
+using Breeze.Thermodynamics: dry_air_gas_constant
+using CUDA
+
+# ## Thermodynamic parameters
+#
+# We define the base state with surface pressure ``p_0 = 1000 \, {\rm hPa}``
+# and reference temperature ``T_0 = 300 \, {\rm K}``:
+
+constants = ThermodynamicConstants()
+g = constants.gravitational_acceleration
+cᵖᵈ = constants.dry_air.heat_capacity
+Rᵈ = dry_air_gas_constant(constants)
+
+p₀ = 100000                 # Pa - surface pressure
+T₀ = 300                    # K - reference temperature
+θ₀ = T₀                     # K - reference potential temperature
+U  = 20                     # m s⁻¹ - uniform background wind
+
+# Derived atmospheric parameters:
+
+N² = g^2 / (cᵖᵈ * T₀)       # s⁻² - Brunt–Väisälä frequency squared
+N  = sqrt(N²)               # s⁻¹ - Brunt–Väisälä frequency
+β  = g / (Rᵈ * T₀)          # m⁻¹ - density scale parameter
+
+# ## Schär mountain parameters
+#
+# The mountain profile parameters following [Schar2002](@cite):
+
+h₀ = 250                    # m - peak mountain height (use 25 m for strict linearity)
+a  = 5000                   # m - Gaussian half-width parameter
+λ  = 4000                   # m - terrain corrugation wavelength
+K  = 2π / λ                 # rad m⁻¹ - terrain wavenumber
+
+# ## Grid configuration
+#
+# The domain spans 200 km horizontally (±100 km) and 20 km vertically. We use a non-uniform
+# vertical grid with exponential refinement near the surface to resolve the terrain,
+# transitioning to uniform 500 m spacing above 1 km altitude.
+
+Nx, Nz = 200, 100
+L, H = 100kilometers, 20kilometers
+
+# Vertical grid stretching parameters:
+
+z_transition = 1000         # m - transition height to uniform spacing
+dz_top = 500                # m - constant spacing above transition
+
+# Calculate grid distribution:
+
+Nz_top = ceil(Int, (H - z_transition) / dz_top)     # cells in uniform region
+Nz_bottom = Nz - Nz_top                             # cells in stretched region
+
+# Construct hybrid vertical grid:
+
+z_stretched = ExponentialDiscretization(Nz_bottom, 0, z_transition, scale = z_transition / 8, bias=:left)
+z_uniform = range(z_transition + dz_top, H; length=Nz_top)
+z_faces = vcat(z_stretched.faces, collect(z_uniform))
```

Comment:
This is printing the entire array: https://numericalearth.github.io/BreezeDocumentation/previews/PR333/literated/two_dimensional_mountain_wave/#Grid-configuration
```suggestion
z_faces = vcat(z_stretched.faces, collect(z_uniform))
nothing #hide
```